protocol/genesis/ssv/validator/msgqueue_consumer.go

```diff
@@ -142,6 +142,15 @@ func (v *Validator) ConsumeQueue(logger *zap.Logger, msgID genesisspectypes.Mess
 			logger.Error("❗ got nil message from queue, but context is not done!")
 			break
 		}
+
+		slot, err := msg.Slot()
+		if err != nil {
+			logger.Error("❗ can't get slot from msg", zap.Error(err))
```

Comment:
The error text is different in all error checks after `msg.Slot()`, can you please unify the error text to make it more searchable?

Reply:
fixing, thanks.